backend/app/routes/markets.js

```diff
@@ -76,4 +77,43 @@ router.get("/quote/:symbol", async (req, res) => {
   }
 });
 
+// get info for banner
+// TODO: cache results (redis?) to prevent excessive api calls on production
+router.get("/promo", async (req, res) => {
+  try {
+    const gainers = await iex.list("gainers");
+    const losers = await iex.list("losers");
+    const mostactive = await iex.list("mostactive");
+    const volume = await iex.list("iexvolume");
+    const percent = await iex.list("iexpercent");
+
+    let collection = [
+      ...gainers,
+      ...losers,
+      ...mostactive,
+      ...volume,
+      ...percent,
+    ];
+
+    let hashmap = {};
```

Comment:
Can we use a more meaningful varibale name for this? 😬

Reply:
good point: https://github.com/ngambino0192/stocks/pull/12/commits/e42c0ea98c9517467363d22d3f18af1256b26dce